controllers/gatekeeper_controller.go

```diff
@@ -27,6 +27,7 @@ import (
 	admregv1 "k8s.io/api/admissionregistration/v1"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This package is already imported in the line above.

Reply:
Done